packages/@vue/cli-plugin-e2e-nightwatch/ui.js

```diff
@@ -9,10 +9,22 @@ module.exports = api => {
         type: 'input',
         default: '',
         description: 'nightwatch.tasks.test.url'
+      }, {
+        name: 'config',
+        type: 'input',
+        default: '',
+        description: 'Use custom nightwatch config file (overrides internals)'
```

Comment:
Those descriptions need to be localized with `packages/@vue/cli-ui/locales/en.json`. See the other prompts above for examples.